### PR TITLE
Added: README comment that Windows containers not yet supported in vNet

### PR DESCRIPTION
**Pull request by james-flynn-ie**
Windows containers are not yet supported in a container group in a Virtual Net (https://docs.microsoft.com/en-us/azure/container-instances/container-instances-virtual-network-concepts#other-limitations)

Adding this note as it cost me some time to implement the module and then receive the message that it isn't supported at present.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ terraform destroy
 
 #### Terraform ACI DevOps Agents - Deploy Linux agents in an existing virtual network
```
Comment (jcorioland) [inline README.md:73]:
```suggestion
*Note: Virtual Network integration is only supported for Linux Containers in ACI. This part [does not apply to Windows Containers](https://docs.microsoft.com/en-us/azure/container-instances/container-instances-virtual-network-concepts#other-limitations).*
```

File: README.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ terraform destroy
...
 
-The configuration below can be used to deploy Azure DevOps agents in Linux containers, in an existing virtual network.
+The configuration below can be used to deploy Azure DevOps agents in Linux containers, in an existing virtual network. Windows containers are not yet supported in a container group deployed to a Virtual Network.
```
Comment (jcorioland) [inline README.md:74]:
```suggestion
The configuration below can be used to deploy Azure DevOps agents in Linux containers, in an existing virtual network.
```

**Review by jcorioland — CHANGES_REQUESTED** [review]
Hi @james-flynn-ie - thank you for your contribution. As the entire section is titled "Deploy Linux agents" I suggest we had a note in the begining to make clear it apply only to Linux. Instead of at the end of the paragraph like you suggested.
Once you apply the suggested changes, if you accept, I'll merge the PR :-)

**Comment by james-flynn-ie** [comment]
Thank you for the suggested changes @jcorioland, they've been committed now. :+1:

**Review by jcorioland — APPROVED** [review]

**Pull request merged**